addons/website_hr_recruitment/views/website_hr_recruitment_templates.xml

```diff
@@ -285,10 +285,16 @@
                         <div t-if="job.address_id" class="d-flex flex-column align-items-baseline">
                             <span class="text-muted small">Location</span>
                             <h6 t-field="job.address_id" t-options='{
+                                "widget": "contact",
                                 "fields": ["city"],
-                                "no_tag_br": True
+                                "no_tag_br": True,
+                                "no_marker": True
                             }'/>
                         </div>
+                        <div t-if="not job.address_id" class="d-flex flex-column align-items-baseline">
+                            <span class="text-muted small">Location</span>
+                            <h6>Remote</h6>
+                        </div>
```

Comment:
Just noticed that this could be simplified:
```xml
<div class="d-flex flex-column align-items-baseline">
    <span class="text-muted small">Location</span>
    <h6 t-if="job.address_id" t-field="job.address_id" t-options='{
        "widget": "contact",
        "fields": ["city"],
        "no_tag_br": True,
        "no_marker": True
    }'/>
    <h6 t-else="">Remote</h6>
</div>
```